mitodl/micromasters
# Added check for missing migrations

#### What are the relevant tickets?
Fixes #1461

#### What's this PR do?
Runs migrations with `--fake` enabled to see if there's a warning message about migrations not matching `models.py`. The build should fail if there is a migration missing

#### How should this be manually tested?
Make a minor change to some field in a model, then run `./detect_missing_migrations.sh` in Docker. Run `echo $?` to see the return value of the last command. You should get `1`. Revert your change to the field, then run `./detect_missing_migrations.sh` again, then run `echo $?`. You should get `0` now.
